client/src/index.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import InfiniteScroll from 'react-infinite-scroller';
+import axios from 'axios'
+
+// ReactDOM.render(<App />, document.getElementById('root'));
+
+// // If you want your app to work offline and load faster, you can change
+// // unregister() to register() below. Note this comes with some pitfalls.
+// // Learn more about service workers: https://bit.ly/CRA-PWA
+// serviceWorker.unregister();
+
+
+const UnsplashImage = ({ url, key }) =>
+React.createElement("div", { className: "image-item", key: key },
+React.createElement("img", { src: url }));
+
+
+
+let Collage = () => {
+  const [images, setImages] = React.useState([]);
+  const [loaded, setIsLoaded] = React.useState(false);
+
+  React.useEffect(() => {
+    fetchImages();
+  }, []);
+
+  const fetchImages = (count = 10) => {
+    const apiRoot = "https://api.unsplash.com";
+    const accessKey =
+    "a22f61e98da4efa25d8860e77a91a596867dd335ecdf7feb12e086943db9565a";
+
+    axios.
+    get(`${apiRoot}/photos/random?client_id=${accessKey}&count=${count}`).
+    then(res => {
+      setImages([...images, ...res.data]);
+      setIsLoaded(true);
+
+      console.log(images);
+    });
+  };
+
+  return (
+    React.createElement("div", { className: "hero is-fullheight is-bold is-info" },
```

Comment:
Why are you using .createElement instead of jxs syntax
ie: <div className="hero is-fullheight is-bold is-info></div>

Reply:
Is this a transpiled file?